components/utils/MeasureUploadHeader.tsx

```diff
@@ -0,0 +1,45 @@
+import { Text, Popover, List, Anchor, ActionIcon } from '@mantine/core';
+import React, { useState } from 'react';
+import { InfoCircle } from 'tabler-icons-react';
+
+export default function MeasureUploadHeader() {
+  const [opened, setOpened] = useState(false);
+
+  return (
+    <div
+      style={{
+        display: 'flex',
+        alignItems: 'center',
+        height: '100%',
+        width: '100%',
+        padding: 20
+      }}
+    >
+      <div style={{ paddingRight: 5 }}>
+        <Text size="xl">Step 1: Upload a Measure Bundle</Text>
+      </div>
+      <div>
+        <Popover
+          opened={opened}
+          onClose={() => setOpened(false)}
+          target={
+            <ActionIcon aria-label={'More Information'} onClick={() => setOpened(o => !o)}>
+              <InfoCircle size={20} />
+            </ActionIcon>
+          }
+        >
+          The uploaded Measure Bundle must contain:
+          <List>
+            <List.Item>one measure resource</List.Item>
+            <List.Item>all dependent library resources used</List.Item>
+            <List.Item>
+              all ValueSets needed for measure calculation see{' '}
+              <Anchor href="https://github.com/projecttacoma/fqm-testify#adding-test-cases">here</Anchor> for more info
```

Comment:
This links to the fqm-testify README for adding test cases. I assumed you wanted a link to fqm-execution instead?

Reply:
Ah so I guess it won't appear when you are testing because I made the changes to the README in this PR but it will link to that section and it will then include a section on adding ValueSets to the measure bundle. Does that make sense?

Reply:
Got it. You probably want it to link to `#adding-valuesets-to-fhir-measure-bundle` then so it goes directly to that section. It's fine if you push up a link that doesn't 100% work yet since it'll work when this is merged